app/controllers/hyrax/batch_edits_controller.rb

```diff
@@ -35,11 +35,19 @@ def check_for_empty!
     end
 
     def destroy_collection
+      # This flag is especially useful to catch non empty admin sets.
+      success = true
       batch.each do |doc_id|
         obj = ActiveFedora::Base.find(doc_id, cast: true)
-        obj.destroy
+        success = false if obj.destroy == false
```

Comment:
Why are we checking for `obj.destroy == false`?  What could cause this situation?  Do you have a test for this change?